### PR TITLE
openvino: added 2023.3.0 version

### PR DESCRIPTION
**Pull request by RaguzovaTatyana**
Specify library name and version: openvino/2023.3.0

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/openvino/all/patches/2023.3.0/0001-git-version.patch
Comment (ilya-lavrenov) [inline recipes/openvino/all/patches/2023.3.0/0001-git-version.patch:1]: this patch is not required, because it's already part of the product

**Comment by ilya-lavrenov** [comment]
Hi @RubenRBS 
Could you please help here? What is wrong with `c3i/conan-v2/pr-merge` ? It's red for a while for current PR and no way to observe logs

**Comment by uilianries** [comment]
Hello @ilya-lavrenov ! The logs of the last failed build should be available at https://c3i.jfrog.io/c3i/misc-v2/summary.html?json=https://c3i.jfrog.io/c3i/misc-v2/logs/pr/22244/5-windows-msvc/openvino/2023.1.0//summary.json

The link is from the previous comment https://github.com/conan-io/conan-center-index/pull/22244#issuecomment-1905468385

**Comment by RaguzovaTatyana** [comment]
Hi, @RubenRBS, @uilianries!
Could you please help us figure out why the 'c3i/conan-v2/pr-merge' checks for our older releases are no longer passing with failed protobuf? And how can we fix this?

**Comment by uilianries** [comment]
I just restarted the Conan 2.x build. Let's see if we have a different result now.

**Comment by RaguzovaTatyana** [comment]
@uilianries , looks like nothing's changed.

**Comment by uilianries** [comment]
@RaguzovaTatyana I opened the PR #22542 to revert Protobuf changes.

**Review by RubenRBS — DISMISSED** [review]

**Comment by SpaceIm** [comment]
Logic in openvino recipe is broken and was working by luck. See onnx, opencv, googleapis or grpc recipes for robust logic when you need protobuf or flatbuffers libraries AND executables.

**Comment by uilianries** [comment]
Locally, I have the same result with the current commit [84dda3c](https://github.com/conan-io/conan-center-index/pull/22244/commits/84dda3c0f7368dddaa0a40bd0c69c6664ab52862)

**Comment by uilianries** [comment]
I'll wait for new build result here in this PR. In case not passing for the same reason involving Protobuf, then I'll revert last protobuf change in the PR #22542

**Comment by ilya-lavrenov** [comment]
> I'll wait for new build result here in this PR. In case not passing for the same reason involving Protobuf, then I'll revert last protobuf change in the PR #22542

@uilianries I've applied suggestion provided by @SpaceIm in the PR https://github.com/conan-io/conan-center-index/pull/22289
Looks like v2 build has passed now.

Suggest merging that PR first, while this one will just add new OpenVINO (as it was originally supposed :) )

**Comment by uilianries** [comment]
@ilya-lavrenov Thank you for checking it!

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by uilianries** [comment]
Hello @RaguzovaTatyana, we are facing an internal error with Mac machines. But it should be solved now. I'll restart this PR now.

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by ilya-lavrenov** [comment]
@jwillikers please, have a look

**Review by franramirez688 — APPROVED** [review]

**Pull request merged**